### PR TITLE
Setup Github actions for commcare-android PRs

### PR DESCRIPTION
**Pull request by avazirna**
## Summary
This PR enables Github Actions for commcare-android PRs.

### CODE REVIEW

**Review by shubham1g5 — APPROVED** [review]
Looks good, though confirming this doesn't currently run the instrumentation tests on Borwserstack ?

**Comment by avazirna** [comment]
> Looks good, though confirming this doesn't currently run the instrumentation tests on Borwserstack ?

Yes, it doesn't, that will be the next step.

**Comment by shubham1g5** [comment]
@avazirna thanks! Another question, have we added other secrets [listed here](https://github.com/dimagi/commcare-android/blob/master/app/build.gradle#L138) to this workflow ?

File: .github/workflows/commcare-android-pr-workflow.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
+name: commcare-android PR CI
+
+on:
+  pull_request:
+    types:
+      - created
+  issue_comment:
```
Comment (shubham1g5) [inline .github/workflows/commcare-android-pr-workflow.yml:7]: This is triggering a build on any comment we make on PR. We should remove it or make it trigger on a specific comment like 'retest this please' if possible.

Comment (avazirna) [reply .github/workflows/commcare-android-pr-workflow.yml:7]: yes, there is a [PR](https://github.com/dimagi/commcare-android/pull/2644) to fix this already.

**Pull request merged**